src/components/BounceRuleDetailed/Changelog/index.jsx

```diff
@@ -7,45 +7,97 @@ import {
   TableHeader,
   TableRow,
 } from "@sendgrid/ui-components/table/table";
-import mockChangelog from "../../../mocks/index";
+import Card from "@sendgrid/ui-components/card";
+import { CSVLink } from "react-csv";
+import moment from "moment";
+import { Row } from "../../Row";
+import { Column } from "../../Column";
 
-const Changelog = ({ handleButtonClicked }) => (
-  <div>
-    <h2>Changelog</h2>
-    <Table>
-      <TableHeader>
-        <TableRow>
-          <HeaderCell>Date</HeaderCell>
-          <HeaderCell>User</HeaderCell>
-          <HeaderCell>Commit Message</HeaderCell>
-          <HeaderCell className="actions-align">Actions</HeaderCell>
-        </TableRow>
-      </TableHeader>
-      <TableBody>
-        {mockChangelog.map(change => (
-          <ChangelogMin
-            key={change.id}
-            change={change}
-            handleButtonClicked={handleButtonClicked}
-          />
-        ))}
-      </TableBody>
-    </Table>
+import { WriteSelectors } from "../selectors";
+
+function showIndividualChanges(
+  changelog,
+  changelogLimit,
+  handleChangelogClicked
+) {
+  return changelog
+    .slice(0, changelogLimit)
+    .map((change, index) => (
+      <IndividualChange
```

Comment:
Hmm IndividualChange is the same as Change and it feels weird to say individualchanges. Can refactor the naming to be more concise like ```<Change ... />``` and ```showChanges()```